uwsgi.h

```diff
@@ -1515,8 +1522,9 @@ struct wsgi_request {
 	// when set, do not send warnings about bad behaviours
 	int post_warning;
 
-	size_t range_from;
-	size_t range_to;
+	enum uwsgi_range range_parsed;
+	int64_t range_from;
```

Comment:
On one side, I really care a little, cause our company doesn't use 32bit platform.

On the other side, this offsets are read from user input, and there is application/plugin decision how to treat them.
For example `plugin/rados` could be used to fetch part of a huge file, and `rados_read` has signature:
```
int rados_read(rados_ioctx_t io, const char * oid, char * buf, size_t len, uint64_t off)
```
So, it uses 64bit offsets, and there is no reason to limit user.